apps/emqx_connector_aggregator/src/emqx_connector_aggregator.erl

```diff
@@ -99,10 +99,12 @@ write_records_limited(Name, Buffer = #buffer{max_records = MaxRecords}, Records)
             write_records(Name, Buffer, Records)
```

Comment:
I guess you can just pass down `NWritten` from here, to skip going into ETS for basically the same number.